conda/recipes/ucx-py/meta.yaml

```diff
@@ -0,0 +1,52 @@
+{% set name = 'ucx-py' %}
+{% set version = environ.get('GIT_DESCRIBE_TAG', '0.0.0.dev').lstrip('v') + environ.get('VERSION_SUFFIX', '') %}
+{% set py_version = environ.get('CONDA_PY', '38') %}
+{% set cuda_version = '.'.join(environ.get('CUDA', '11.2').split('.')[:2]) %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  path: ../../..
+
+build:
+  number: {{ GIT_DESCRIBE_NUMBER }}
+  string: py{{ py_version }}_g{{ GIT_DESCRIBE_HASH }}_{{ GIT_DESCRIBE_NUMBER }}
+  script_env:
+    - VERSION_SUFFIX
+    - CC
+    - CXX
+    - CUDAHOSTCXX
+  script:
+    - {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - python
+    - pip
+    - cython >=0.29.14,<3.0.0a0
+    - libhwloc >=2.3.0,<2.4.0a0
+    - ucx
```

Comment:
does this line need to be `ucx {{ ucx }}` in order to use the value from `conda_build_config`? Or is it optional?

Reply:
No conda-build automatically handles that